desktop/components/main_layout/templates/index.jade

```diff
@@ -1,6 +1,6 @@
 //- Override any locals with `append locals`
 block locals
-  - bodyClass = helpers ? helpers.buildBodyClass(sd, 'body-header-fixed') : '';
+  - bodyClass = helpers && helpers.buildBodyClass ? helpers.buildBodyClass(sd, 'body-header-fixed') : '';
```

Comment:
can do `helpers && helpers.buildBodyClass(sd, 'body-header-fixed') || ''` -- but why would `helpers` not be here?